pkgs/servers/headscale/default.nix

```diff
@@ -7,41 +7,30 @@
 }:
 buildGoModule rec {
   pname = "headscale";
-  version = "0.22.3";
+  version = "0.23.0";
 
   src = fetchFromGitHub {
     owner = "juanfont";
     repo = "headscale";
     rev = "v${version}";
-    hash = "sha256-nqmTqe3F3Oh8rnJH0clwACD/0RpqmfOMXNubr3C8rEc=";
+    hash = "TODO";
   };
 
-  vendorHash = "sha256-IOkbbFtE6+tNKnglE/8ZuNxhPSnloqM2sLgTvagMmnc=";
-
-  patches = [
-    # backport of https://github.com/juanfont/headscale/pull/1697
-    ./trim-oidc-secret-path.patch
-
-    # fix for headscale not reacting to SIGTERM
-    # see https://github.com/juanfont/headscale/pull/1480 and https://github.com/juanfont/headscale/issues/1461
-    ./sigterm-fix.patch
-  ];
+  vendorHash = "TODO";
 
   ldflags = ["-s" "-w" "-X github.com/juanfont/headscale/cmd/headscale/cli.Version=v${version}"];
 
   nativeBuildInputs = [installShellFiles];
   checkFlags = ["-short"];
 
-  tags = ["ts2019"];
-
   postInstall = ''
```

Comment:
```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Reply:
What does it do?

Reply:
Fix cross compilation.

Reply:
When cross compiling we cannot execute the binary to generate the files. Alternatives would be to use an emulator or compile the binary twice.